crates/forge/src/scarb.rs

```diff
@@ -221,6 +223,41 @@ pub fn dependencies_for_package(
     Ok(dependencies)
 }
 
+fn validate_fork_config(config: Result<ForgeConfig>) -> Result<ForgeConfig> {
```

Comment:
:50_50: because we only validate fork configs 😛

Reply:
True, but it can be expanded in more general function later on

Reply:
It validates the config from Scarb.toml, the one from fuzzing PR validates command line arguments

Reply:
I think we should consolidate that in some way

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/714

Reply:
Not sure if this can be consolidated except for some core logic. The `validate` we have in fuzzing is technically also a custom parsers for the command line argument.